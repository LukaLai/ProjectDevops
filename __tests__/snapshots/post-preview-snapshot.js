import { render } from '@testing-library/react'
import { PostPreview } from '../../src/app/_components/post-preview'

it('renders homepage unchanged', () => {
    const { container } = render(<PostPreview date={'2024-04-17T05:35:07.322Z'} title={'test'} author={'jj'} coverImage={'/public/assets/blog/authors/jj.jpeg'} slug={'slug'} excerpt={'excerpt'}/>)
    expect(container).toMatchSnapshot()
})