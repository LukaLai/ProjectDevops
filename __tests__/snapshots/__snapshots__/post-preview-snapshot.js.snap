// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders homepage unchanged 1`] = `
<div>
  <div>
    <div
      class="mb-5"
    >
      <div
        class="sm:mx-0"
      >
        <a
          aria-label="test"
          href="/posts/slug"
        >
          <img
            alt="Cover Image for test"
            class="shadow-sm w-full hover:shadow-lg transition-shadow duration-200"
            data-nimg="1"
            decoding="async"
            height="630"
            loading="lazy"
            src="/_next/image?url=%2Fpublic%2Fassets%2Fblog%2Fauthors%2Fjj.jpeg&w=3840&q=75"
            srcset="/_next/image?url=%2Fpublic%2Fassets%2Fblog%2Fauthors%2Fjj.jpeg&w=1920&q=75 1x, /_next/image?url=%2Fpublic%2Fassets%2Fblog%2Fauthors%2Fjj.jpeg&w=3840&q=75 2x"
            style="color: transparent;"
            width="1300"
          />
        </a>
      </div>
    </div>
    <h3
      class="text-3xl mb-3 leading-snug"
    >
      <a
        class="hover:underline"
        href="/posts/slug"
      >
        test
      </a>
    </h3>
    <div
      class="text-lg mb-4"
    >
      <time
        datetime="2024-04-17T05:35:07.322Z"
      >
        April	17, 2024
      </time>
    </div>
    <p
      class="text-lg leading-relaxed mb-4"
    >
      excerpt
    </p>
    <div
      class="flex items-center"
    >
      <img
        class="w-12 h-12 rounded-full mr-4"
        data-nimg="1"
        decoding="async"
        height="12"
        src=""
        style="color: transparent;"
        width="12"
      />
      <div
        class="text-xl font-bold"
      />
    </div>
  </div>
</div>
`;
